doc/developers/contributing.rst

```diff
@@ -714,7 +714,7 @@ Building the documentation requires installing some additional packages:
 
     pip install sphinx sphinx-gallery numpydoc matplotlib Pillow pandas \
                 scikit-image packaging seaborn sphinx-prompt \
-                sphinxext-opengraph plotly
+                sphinxext-opengraph plotly pooch
```

Comment:
Interesting. Why do we need `pooch`? Is it only an optional dependency in `scipy`?

Reply:
I don't know. I had to install it to get rid of errors and warnings.

Reply:
Yep this is due to scipy: https://docs.scipy.org/doc/scipy/reference/datasets.html#how-dataset-retrieval-and-storage-works